packages/infrastructure/kube_airbyte/main.tf

```diff
@@ -0,0 +1,1386 @@
+    )
+  }
+  statement {
+    effect = "Allow"
+    actions = [
+      "s3:ListBucket"
+    ]
+    resources = concat(
+      [module.airbyte_bucket.bucket_arn],
+      var.connected_s3_bucket_arns
+    )
+  }
+  statement {
+    effect = "Allow"
+    actions = [
+      "s3:GetBucketLocation"
+    ]
+    resources = ["arn:aws:s3:::*"]
+  }
+
+  # Add multipart upload permissions which are needed by the S3 destination
+  statement {
+    effect = "Allow"
+    actions = [
+      "s3:ListMultipartUploadParts",
+      "s3:AbortMultipartUpload",
+      "s3:ListBucketMultipartUploads"
+    ]
+    resources = concat(
+      [module.airbyte_bucket.bucket_arn, "${module.airbyte_bucket.bucket_arn}/*"],
+      flatten([for arn in var.connected_s3_bucket_arns : [arn, "${arn}/*"]])
+    )
+  }
+}
+
+module "aws_permissions" {
+  source = "../kube_sa_auth_aws"
+
+  service_account           = kubernetes_service_account.airbyte_sa.metadata[0].name
+  service_account_namespace = local.namespace
+  iam_policy_json           = data.aws_iam_policy_document.airbyte_bucket.json
+  ip_allow_list             = var.aws_iam_ip_allow_list
+}
+
+/***************************************
+* Secret Configuration
+***************************************/
+
+resource "random_password" "admin_password" {
+  length  = 32
+  special = false
+}
+
+resource "kubernetes_secret" "airbyte_secrets" {
+  metadata {
+    name      = "airbyte-config-secrets"
+    namespace = local.namespace
+    labels    = data.pf_kube_labels.labels.labels
+  }
+
+  data = {
+    "license-key"             = var.license_key # For enterprise edition
+    "instance-admin-email"    = var.admin_email
+    "instance-admin-password" = random_password.admin_password.result
+  }
+}
+
+/***************************************
+* Airbyte Helm Chart
+***************************************/
+
+data "aws_region" "current" {}
+
+resource "helm_release" "airbyte" {
+  namespace       = local.namespace
+  name            = local.name
+  repository      = "https://airbytehq.github.io/helm-charts"
+  chart           = "airbyte"
+  version         = var.airbyte_helm_version
+  recreate_pods   = false
+  atomic          = var.wait
+  cleanup_on_fail = var.wait
+  wait            = var.wait
+  force_update    = false
+  wait_for_jobs   = true
+  max_history     = 5
+
+  lifecycle {
+    precondition {
+      condition     = var.temporal_db_max_conns >= var.temporal_db_max_idle_conns
+      error_message = "temporal_db_max_conns must be greater than or equal to temporal_db_max_idle_conns"
+    }
+  }
+
+  values = [
+    yamlencode({
+      fullnameOverride = "airbyte"
+
+      global = {
+        serviceAccountName = kubernetes_service_account.airbyte_sa.metadata[0].name
+        edition            = var.airbyte_edition
+        airbyteUrl         = "https://${var.domain}"
+
+        image = {
+          tag = var.airbyte_version
+        }
+
+        auth = {
+          enabled = false
```

Comment:
if you provided an enterprise license key, wouldn't you want auth to be enabled?